### PR TITLE
Update dm-worker-configuration-file.md

### PR DESCRIPTION
**Pull request by lance6716**
### First-time contributors' checklist 

- [x] I've signed [**Contributor License Agreement**](https://cla-assistant.io/pingcap/docs-cn) that's required for repo owners to accept my contribution.

### What is changed, added or deleted? (Required)

add more description for https://github.com/pingcap/docs-cn/pull/8113

### Which TiDB version(s) do your changes apply to? (Required)

**Tips for choosing the affected version(s):**

By default, **CHOOSE MASTER ONLY** so your changes will be applied to the next TiDB major or minor releases. If your PR involves a product feature behavior change or a compatibility change, **CHOOSE THE AFFECTED RELEASE BRANCH(ES) AND MASTER**.

For details, see [tips for choosing the affected versions (in Chinese)](https://github.com/pingcap/docs-cn/blob/master/CONTRIBUTING.md#版本选择指南).

- [x] master (the latest development version)
- [x] v5.4 (TiDB 5.4 versions)
- [ ] v5.3 (TiDB 5.3 versions)
- [ ] v5.2 (TiDB 5.2 versions)
- [ ] v5.1 (TiDB 5.1 versions)
- [ ] v5.0 (TiDB 5.0 versions)
- [ ] v4.0 (TiDB 4.0 versions)
- [ ] v3.1 (TiDB 3.1 versions)
- [ ] v3.0 (TiDB 3.0 versions)
- [ ] v2.1 (TiDB 2.1 versions)

### What is the related PR or file link(s)?

- This PR is translated from:
- Other reference link(s): https://github.com/pingcap/tiflow/pull/3985

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

**Review by TomShawn — APPROVED** [review]

File: dm/dm-worker-configuration-file.md [hunk 1]
```diff
@@ -25,6 +25,7 @@ join = "http://127.0.0.1:8261,http://127.0.0.1:8361,http://127.0.0.1:8461"
 
 keepalive-ttl = 60
 relay-keepalive-ttl = 1800 # 版本 2.0.2 新增
+# relay-dir = "relay_log" # 版本 5.4.0 新增
```
Comment (sunzhaoyang) [inline dm/dm-worker-configuration-file.md:28]: 这个 relay_log 是指 绝对路径 /relay_log 还是相对路径？

Comment (lance6716) [reply dm/dm-worker-configuration-file.md:28]: 不写/就是相对路径，相对于部署路径。咋办

Comment (sunzhaoyang) [reply dm/dm-worker-configuration-file.md:28]: 注释说明一下就行，默认应该是在 deploy 目录下吧？

**Review by sunzhaoyang — APPROVED** [review]
LGTM

File: dm/dm-worker-configuration-file.md [hunk 2]
```diff
@@ -46,6 +47,7 @@ cert-allowed-cn = ["dm"]
 | `join` | 对应一个或多个 DM-master 配置中的 [`master-addr`](/dm/dm-master-configuration-file.md#global-配置)。 |
 | `keepalive-ttl` | 当绑定的上游数据源没有启用 relay log 时，DM-worker 向 DM-master 保持存活的周期，单位为秒。默认是 60 秒。 |
 | `relay-keepalive-ttl` | 当绑定的上游数据源启用 relay log 时，DM-worker 向 DM-master 保持存活的周期，单位为秒。默认是 1800 秒。在版本 2.0.2 新增。 |
+| `relay-dir` | 当绑定的上游数据源启用 relay log 时，DM-worker 将 relay log 保存在该路径下。该配置优先级比上有数据源配置更高。在版本 5.4.0 新增。 |
```
Comment (TomShawn) [inline dm/dm-worker-configuration-file.md:50]:
```suggestion
| `relay-dir` | 当绑定的上游数据源启用 relay log 时，DM-worker 将 relay log 保存在该路径下。该配置优先级比上游数据源配置更高。在版本 5.4.0 新增。 |
```

**Review by TomShawn — APPROVED** [review]

**Pull request merged**